src/modules/admin/services/admin.service.ts

```diff
@@ -118,15 +113,22 @@ export class AdminService {
 
   async recoverUserPassword(
```

Comment:
missing the docstrings

Reply:
Yes, I know that. 
Nearly all the functions don't have docstrings. I'll create a separate PR for them, once the Demo flow completes.

Reply:
Perfect